wscript

```diff
@@ -127,10 +137,13 @@ def configure(cfg):
 
     cfg.env.BOARD = cfg.options.board
     cfg.env.DEBUG = cfg.options.debug
+    cfg.env.AUTOCONFIG = cfg.options.autoconfig
```

Comment:
But, yes, you're right. Only the first one is necessary. I wonder why I did this. I'll fix it anyway.